Stack/Opc.Ua.Core/Stack/Types/ContentFilter.Evaluate.cs

```diff
@@ -2109,6 +2163,102 @@ private bool RelatedTo(FilterContext context, IFilterTarget target, ContentFilte
                 return false;
             }
         }
+
+        /// <summary>
+        /// BitwiseAnd FilterOperator
+        /// </summary>
+        private object BitwiseAnd(FilterContext context, IFilterTarget target, ContentFilterElement element)
```

Comment:
I had a second look at this. I think you indeed need to return here not a bool but the result of the operation.
Then if there are multiple filters it can be used as input. However how it is used when its the only operation is unclear, thats then back to the null handling described in the spec.

Reply:
Yes, changed back to returning the values